arbnode/batch_poster.go

```diff
@@ -245,7 +245,7 @@ var DefaultBatchPosterConfig = BatchPosterConfig{
 	DisableDapFallbackStoreDataOnChain: false,
 	// This default is overridden for L3 chains in applyChainParameters in cmd/nitro/nitro.go
 	MaxSize:             100000,
-	MaxEigenDABatchSize: 2_000_000,
+	MaxEigenDABatchSize: 16_000_000,
```

Comment:
not sure if it is intentional to leave some bytes unused. EigenDA supports 16MiB blob, 16777216 bytes

Reply:
[93389d9](https://github.com/Layr-Labs/nitro/pull/36/commits/93389d9b3dd7abc6a287b7fabc75f3bf33ed9b9c)